server/routes/trackReceipt.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const router = express.Router()
+const jwt = require('../utils/jwt')
+const TrackReceipt = require('../models/TrackReceipt')
+
+router.post('/', jwt.verifyJWT, async (req, res, next) => {
+  /* adds track receipt to db */
+  try {
+    const userId = req.userId
+    const { otherUserId, track, username, similarityScore, playlist, addedAt } = req.body
+
+    const receipt = new TrackReceipt({ userId, otherUserId, track, username, similarityScore, playlist, addedAt })
+    await receipt.save()
+    res.status(200).json()
+  } catch (error) {
+    next(error)
+  }
+})
+
+router.get('/', jwt.verifyJWT, async (req, res, next) => {
+  /* returns all track receipts belonging to current user */
+  try {
+    const userId = req.userId
+    const receipts = await TrackReceipt.find({ userId })
+
+    /* delete expired receipts */
+    const promises = receipts.map(async (receipt) => {
+      const receiptDate = new Date(receipt.addedAt)
+      const receiptSeconds = receiptDate.getTime() / 1000
+      const timeNow = Date.now() / 1000
+
+      /* delete if longer than 7 days */
+      if ((timeNow - receiptSeconds) > 604800) {
```

Comment:
604800 is a [magic number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad), move it into a well named variable.